### PR TITLE
direnv: update to 2.30.3.

### PR DESCRIPTION
**Pull request by josegpt**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/direnv/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'direnv'
 pkgname=direnv
-version=2.28.0
+version=2.30.2
```
Comment (paper42) [inline srcpkgs/direnv/template:3]: .3 was released 14 days ago

**Comment by paper42** [comment]
Why did you close this PR?

**Comment by josegpt** [comment]
I closed it by mistake; I will reopen it and push the newest version to this PR.

**Comment by paper42** [comment]
thanks!

**Pull request merged**